src/Arcus.Observability.Telemetry.Serilog/Arcus.Observability.Telemetry.Serilog.csproj

```
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Authors>Arcus</Authors>
+    <Company>Arcus</Company>
+    <Description>Provides capability to use additional telemetry functionality for Serilog in applications</Description>
```

How about this? (Just a suggestion)
```suggestion
    <Description>Provides capability to improve telemetry with Serilog in applications</Description>
```